### DISCUSSION TITLE
replace smart object in layer and save it

### DISCUSSION THREAD

**Discussion by bnkcrypt** [ImageMagick/ImageMagick#6951] [Digital Image Processing]
Hi I am looking for a convert command that will open psd file and replace an object from a specific layer and save it and convert it to jpg .

**Comment by fmw42** [comment]
I do not believe that Imagemagick can deal with smart objects.  An Imagemagick developer might comment further.

**Comment by dlemstra** [comment]
That is correct there is no support for that in this library.
